dces-drc-integration/src/main/java/uk/gov/justice/laa/crime/dces/integration/service/ContributionService.java

```diff
@@ -55,7 +57,9 @@ public boolean processDailyFiles() {
         return updateContributionsAndCreateFile(successfulContributions, failedContributions);
     }
 
-    private void sendContributionsToDrc(List<ConcurContribEntry> contributionsList, Map<String, CONTRIBUTIONS> successfulContributions, Map<String,String> failedContributions){
+
+    @Retry(name=SERVICE_NAME)
+    public void sendContributionsToDrc(List<ConcurContribEntry> contributionsList, Map<String, CONTRIBUTIONS> successfulContributions, Map<String,String> failedContributions){
```

Comment:
Should talking to the DRC (`sendContributionsToDrc`) use the same retry policy as talking to the MAAT_API (`contributionUpdateRequest`)? Maybe rather than have the divide on Contributions/FDCS, it should be on MAAT-API / DRC (i.e. the owner of the system providing the endpoint?)

Reply:
I mean have DRC_SERVICE and MAATAPI_SERVICE instead of ContributionService and FdcService as the Retry service names

Reply:
Yeah, we can see how things pan out in the course of testing/execution. Something to keep an eye on.